conda_smithy/configure_feedstock.py

```diff
@@ -941,7 +941,7 @@ def _get_build_setup_line(forge_dir, platform, forge_config):
         if platform == "win":
             build_setup += textwrap.dedent(
                 """\
-                run_conda_forge_build_setup
+                CALL run_conda_forge_build_setup
```

Comment:
Without using `CALL`, this batch script coming from `conda-forge-ci-setup` might (will?) exit early with errorlevel = 0, misreporting success without even running the recipe.